[2023-03-17T22:06:04.882+0000] {processor.py:153} INFO - Started process (PID=66) to work on /opt/airflow/dags/test/test_forexdag_definition.py
[2023-03-17T22:06:04.891+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/test/test_forexdag_definition.py for tasks to queue
[2023-03-17T22:06:04.893+0000] {logging_mixin.py:137} INFO - [2023-03-17T22:06:04.893+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test/test_forexdag_definition.py
[2023-03-17T22:06:05.008+0000] {logging_mixin.py:137} INFO - [2023-03-17T22:06:04.985+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/test/test_forexdag_definition.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/test_forexdag_definition.py", line 1, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
[2023-03-17T22:06:05.016+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/test/test_forexdag_definition.py
[2023-03-17T22:06:05.055+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/test/test_forexdag_definition.py took 0.179 seconds
[2023-03-17T22:06:35.823+0000] {processor.py:153} INFO - Started process (PID=109) to work on /opt/airflow/dags/test/test_forexdag_definition.py
[2023-03-17T22:06:35.825+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/test/test_forexdag_definition.py for tasks to queue
[2023-03-17T22:06:35.827+0000] {logging_mixin.py:137} INFO - [2023-03-17T22:06:35.826+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/test/test_forexdag_definition.py
[2023-03-17T22:06:35.896+0000] {logging_mixin.py:137} INFO - [2023-03-17T22:06:35.882+0000] {dagbag.py:343} ERROR - Failed to import: /opt/airflow/dags/test/test_forexdag_definition.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/test/test_forexdag_definition.py", line 1, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
[2023-03-17T22:06:35.902+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/test/test_forexdag_definition.py
[2023-03-17T22:06:35.921+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/test/test_forexdag_definition.py took 0.102 seconds
